ldap/servers/plugins/replication/repl5_agmt.c

```diff
@@ -401,6 +401,10 @@ agmt_new_from_entry(Slapi_Entry *e)
             replica_incr_agmt_count(replica);
         }
     }
+    if (!replica) {
+        slapi_log_err(SLAPI_LOG_ERR, repl_plugin_name, "agmt_new_from_entry - Failed to get replica.\n");
+        goto loser;
```

Comment:
Can we add more info about which replication agreement (dn), or some other identifying info?

Reply:
Ok: I am changing it to:
        slapi_log_err(SLAPI_LOG_ERR, repl_plugin_name, 
                      "agmt_new_from_entry - Failed to get replica for agreement %s on replicated suffix %s).\n",
                      slapi_entry_get_dn(e), tmpstr ? tmpstr : "<NULL>");